docs/remarkable/device.md

```diff
@@ -0,0 +1,75 @@
+There are several ways to make it work, choose whatever works for you
+
+# Automatic
```

Comment:
Since the package is also in toltec's testing brach, it probably would make sense to add information for that way of installing and managing it as well.

Reply:
````suggestion
# Automatic
## Variant 1
Install using [toltec](https://toltec-dev.org/).
```commandline
opkg install rmfakecloud-proxy
rmfakecloudctl set-upstream <URL>
rmfakecloudctl enable
```

## Variant 2
````

Reply:
Please give explicit names instead of "Variant X". Like:

```
# Automatic
## With toltec
## With rmfakecloud-proxy script

# Manual
## Variant 1: Installing a proxy on devices
## Variant 2: With hosted reverse-proxy
## Variant 3: With hosted reverse-proxy and DNS overwriting
```

Reply:
Thans @Eeems for the toltec tip, I added it into the changed and also used @nemunaire suggestion to name the variants accordingly. Btw, I refrained to use "with hosted reverse-proxy", since the very next line says "Connect to the host directly, **without a reverse proxy**, with HTTPS on :443". I am guessing the reverse proxy part should just be a proxy, but I am still trying to figure out how to set it up for my server configuration and what is needed.